engine/src/flutter/shell/platform/linux/fl_renderer.cc

```diff
@@ -388,6 +417,53 @@ guint32 fl_renderer_get_fbo(FlRenderer* self) {
   return 0;
 }
 
+struct _FlRendererCallBlockingTrampolineData {
+  FlRenderer* renderer;
+  gboolean* finished;
+  void (*callback)(gpointer data);
+  gpointer data;
+};
+
+static void fl_renderer_call_blocking_trampoline(gpointer user_data) {
```

Comment:
I found the name of this function a bit confusing, perhaps something like `fl_renderer_call_blocking_on_main_thread`?